src/App.jsx

```diff
@@ -14,70 +16,75 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
+  const heading = (
     <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+      {selectedGood
+        ? `${selectedGood} is selected`
+        : 'No goods selected'
+      }
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
+      {selectedGood
+        && (
+          <button
+            onClick={() => setSelectedGood('')}
```

Comment:
```suggestion
            onClick={clearGood}
```
It's better to create separate functions to handle events. This way, your code will be easier to maintain, extend, and reuse.

Imagine you'll have a situation where you need to reuse the same functionality on another event. Then you'd be able to just use the same function you've created. Also, if you need to extend the functionality of event handling in the future, it'd be easier to do so if you have a separate function. Moreover, if this function's used more than once you'll need to add new functionality only in one function and then call it wherever you need to.